tex/gsp-default.tex

```diff
@@ -210,8 +210,12 @@
 \grecreatedim{initialraise}{0 cm}{scalable}%
 % Space between lines in the annotation
 \grecreatedim{annotationseparation}{0.05cm}{scalable}%
-% Amount to raise (positive) or lower (negative) the annotations from the default position (base line of top annotation aligned with top line of staff)
+% Amount to raise (positive) or lower (negative) the annotations from the default position
 \grecreatedim{annotationraise}{0cm}{scalable}%
+% Space between lines in the commentary
+\grecreatedim{commentaryseparation}{0.05cm}{scalable}%
+% Amount to raise (positive) or lower (negative) the commentary from the default position (base line of bottom commentary aligned with top line of staff)
+\grecreatedim{commentaryraise}{0.4cm}{scalable}%
```

Comment:
Sorry, I have two final comments:

- maybe `commentaryraise` could be smaller, like 2 or 3mm. The Antiphonale Monasticum has commentaries close to the lines, and it really looks good! 
- it could be made easier to rise the commentary for one score only, with an optional argument to `\grecommentary` (as I did in fix-662)